jsonnet/kube-prometheus/addons/aws-vpc-cni.libsonnet

```diff
@@ -40,6 +44,8 @@
         namespace: $.values.kubernetesControlPlane.namespace,
         labels: {
           'app.kubernetes.io/name': 'aws-node',
+          'app.kubernetes.io/component': 'prometheus',
```

Comment:
```suggestion
          'app.kubernetes.io/component': 'cni',
```

Reply:
Ideally we should update the service monitor's matchLabels to reflect the new service's labels.